script/upgrades/dependencies.json

```diff
@@ -1,10 +1,10 @@
 {
   "42220": {},
   "62320": {
-    "USDCet": "0xfD46b16f86449b3e03e516A3632e1919A5CA1A53",
-    "MockUSDCet": "0x5CB8FB4cF7247de5fD514243d72B5e90C1b7BAb7"
+    "MockUSDCet": "0x5CB8FB4cF7247de5fD514243d72B5e90C1b7BAb7",
+    "USDCUSDRateFeedAddr": "0xA1A8003936862E7a15092A91898D69fa8bCE290c"
```

Comment:
How was this address generated?

Reply:
This is the identifier address for `USDCUSD` that is currently used for the sortedOracles. Since we already passed a proposal with this identifier, calculating this in the code with a function seems dangerous, as that could have a bug and end up using a different address from what we have on-chain.